php_phongo.c

```diff
@@ -1152,6 +1152,32 @@ static bool php_phongo_apply_options_to_uri(mongoc_uri_t *uri, bson_t *options T
 
 			continue;
 		}
+
+		if (!strcasecmp(key, MONGOC_URI_GSSAPISERVICENAME)) {
+			bson_t dummy, properties;
```

Comment:
`tmp` would be a more kosher name than `dummy`. We don't need to insult our structs.

`properties` is also used below without being initialized. I'm imagine most compilers will allocate a stack value to zero, but it's technically undefined. Initializing `properties` to `BSON_INITIALIZER` seems proper.

Reply:
I tend to use "dummy" if the variable isn't used at all. It's used as option 8 at http://www.dictionary.com/browse/dummy

Will fix them though :-)